packages/core/src/http-middlewares/after/prepare-response.js

```diff
@@ -8,7 +8,8 @@ const prepareRawResponse = (resp, request) => {
   // TODO: if !2xx should we go ahead and get response.content for them?
   // retain the response signature for raw control
   const extendedResp = {
-    request: request
+    request: request,
+    skipThrowForStatus: request.skipThrowForStatus
```

Comment:
Allows setting the new flag when initiating a request instead of only from `afterResponse`.